gtest/src/system.rs

```diff
@@ -28,6 +28,17 @@ use gear_core::{ids::CodeId, message::Dispatch};
 use path_clean::PathClean;
 use std::{borrow::Cow, cell::RefCell, env, fs, io::Write, path::Path, thread};
 
+/// The testing environment.
+///
+/// ```
+/// use gtest::System;
+///
+/// // Create a new testing system.
```

Comment:
for what and what it keeps, what simulates (the whole mock of chain and its transition)?
and state somewhere that the real on-chain execution may be different (atm its not as up to date as we would like it to be)